### ISSUE TITLE
Moving focus to cookie options menu

### ISSUE THREAD

**Issue by peteredwards** [uol-library/spacefinder-ui#26]
Certainly, on iOS with VoiceOver you cannot move focus to the cookie options menu (I think you get to it via ‘learn more’ or similar). Perhaps worth switching to the Design System version here, as this has been recently re-done and passed a third-party audit.

**Comment by peteredwards** [comment]
I've updated the cookieconsent plugin to the latest version - needs re-testing
